cmd/url.go

```diff
@@ -50,24 +53,52 @@ The created URL can be used to access the specified component from outside the O
 		checkError(err, "")
 		projectName := project.GetCurrent(client)
 
+		var app string
 		var componentName string
+		var urlName string
+
+		if urlListApplication == "" {
+			app, err = application.GetCurrent(client)
+			checkError(err, "")
+		} else {
+			app = urlListApplication
+		}
+
+		if len(urlListComponent) == 0 {
```

Comment:
why is this called `urlListComponent`? Are we reusing the same variable for flags in `list` and `create`?

Reply:
yes

Reply:
> yes, should I add a new one?

renaming should be fine. The current name is really misleading.